### ISSUE TITLE
Legge inn mulighet for støttemedlem

### ISSUE THREAD

**Issue by Vegandi** [Romerike-Jujutsu-Klubb/rjjk_web#263]
Info om støttemedlemsskap, til kr 150,- per år, vipps og konto.

**Comment by donv** [comment]
Dette er vel en oppgave for kasserer eller medlemsansvarlig? @saxofonia ?

Kunne et støttemedlemskap være kr 50 per måned?

**Comment by Vegandi** [comment]
Jeg tenkte 150,- i året, jeg. Send på vipps, liksom, gjør det enkelt.

Alt godt,

Ellen Skjold Kvåle

Denne meldinga er sendt fra en mobiltelefon.

tir. 29. mar. 2022, 17:39 skrev Uwe Kubosch ***@***.***>:

**Comment by donv** [comment]
Enkelt er bra 😄 .

**Comment by saxofonia** [comment]
Hei!

Vi har definert et støttemedlemskap i IMS til kr. 50,-, og har til nå blitt
brukt (tenkt) for støttemedlemmer som tar et styreverv. Jeg tror Ellen
tenker på støttemedlemskap for å støtte økonomisk?
Dette kan dere jo diskutere i styret? Jeg har jo ikke noe med det å gjøre,
jeg. Trenger bare å instrueres i fht. IMS og medlemsweb.



Med vennlig hilsen
*Svein Robert Rolijordet*

Mail: ***@***.***
Web: http://resvero.com/

Mobil: +47 97534766



tir. 29. mar. 2022 kl. 18:07 skrev Ellen Skjold Kvåle <
***@***.***>:

**Comment by Vegandi** [comment]
Ja, jeg tenkte på 150,- som et støttemedlemskap bare for å sende noen
småkroner til klubben. Foreldre og sånn kunne oppfordres til å vippse en
femtilapp.

Alt godt,

Ellen

--
Ellen Skjold Kvåle
Solvegen 18C
2005 RÆLINGEN
Telefon: +47 920 95 703 <https://aidbox.org/ct/?i=525466000001511&t=1>


ons. 30. mar. 2022 kl. 11:45 skrev saxofonia ***@***.***>:
